sql/hive-thriftserver/src/main/java/org/apache/hive/service/CookieSigner.java

```diff
@@ -81,8 +81,7 @@ public String verifyAndExtract(String signedStr) {
       LOG.debug("Signature generated for " + rawValue + " inside verify is " + currentSignature);
     }
     if (!MessageDigest.isEqual(originalSignature.getBytes(), currentSignature.getBytes())) {
-      throw new IllegalArgumentException("Invalid sign, original = " + originalSignature +
-        " current = " + currentSignature);
+      throw new IllegalArgumentException("Invalid sign");
```

Comment:
Could we add` incorrect signature`? `Invalid sign` makes confusion.

Reply:
I went for minimal change here. This is copied from Hive and meant to be a stopgap until Hive addresses an ongoing discussion about a potential security issue here